README.md

```diff
@@ -0,0 +1,46 @@
+## Building Multiarch r-repo image
+
+### Using Manifest 
+
+This proved to be the most efficient way as qemu has some serious lag issues with building multiarch packages.
+
+On armd64 v8 system:
+```
+docker build -t mcneilco/acas-r-repo:1.13.7-linuxarm64v8 -f Dockerfile.repo .
+
+docker push mcneilco/acas-r-repo:1.13.7-linuxarm64v8
+```
+
+On amd64 system:
+```
+docker build -t mcneilco/acas-r-repo:1.13.7-linuxamd64 -f Dockerfile.repo .
+
+docker push mcneilco/acas-r-repo:1.13.7-linuxamd64
+```
+
+On either system:
+
+```
+docker manifest create \
+mcneilco/acas-r-repo:1.13.7 \
+--amend mcneilco/acas-r-repo:1.13.7-linuxamd64 \
+--amend mcneilco/acas-r-repo:1.13.7-linuxarm64v8
+
+docker manifest push mcneilco/acas-r-repo:1.13.7
```

Comment:
This is actually how I built the acas-r-repo:1.13.7 image. It was too slow to compile all the R packages on github actions infrastructure.   I build the amd64 image on my intel mac and the arm64 on my M1 mac

Reply:
Ok. Good to have the workaround documented. ~But it looks like you are also configuring GitHub Actions in this PR. Will we be disabling those since they time out?~

I understand now. There are 2 Dockerfiles, 2 repos. acas-r-repo will change infrequently and needs to be built manually right now. racas-oss will change more frequently and can be built with GitHub actions in a reasonable time, assuming the appropriate acas-r-repo base image already exists. Is that right?

Reply:
That's correct.